### ISSUE TITLE
website blocking not working for linux (ubuntu) version 0.6.2

### ISSUE THREAD

**Issue by henningway** [danielng01/Iris-Builds#2]
just tried but the websites listed (as well as added) may still be accessed.

**Comment by danielng01** [comment]
Yes I know. It works only on Windows. I can make it for Linux. Do you need this feature? :)

**Comment by henningway** [comment]
Yes, I am looking for a good solution to block some pages some of the time. Would prefer an open source or one time buying solution though.
Great design and combination of features, keep it up. :)

On June 5, 2016 11:51:49 AM GMT+02:00, Daniel Georgiev <notifications@github.com> wrote:
>Yes I know. It works only on Windows. I can make it for Linux. Do you
>need this feature? :)
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/danielng01/Iris-Builds/issues/2#issuecomment-223803080

-- 
Diese Nachricht wurde von meinem Android-Mobiltelefon mit K-9 Mail gesendet.
